cli/command/formatter/disk_usage.go

```diff
@@ -65,7 +65,7 @@ reclaimable: {{.Reclaimable}}
 }
 
 func (ctx *DiskUsageContext) Write() (err error) {
-	if ctx.Verbose == false {
+	if !ctx.Verbose {
```

Comment:
Is one of the linters objecting to `ctx.Verbose == false`? I would personally probably write `!ctx.Verbose`, but in my view either are valid, and I certainly don't see potential for bugs or maintainability problems by using one over the other.